extras/rootless/containerd-rootless-setuptool.sh

```diff
@@ -29,19 +29,15 @@ set -eu
 
 # utility functions
 INFO() {
-	# https://github.com/koalaman/shellcheck/issues/1593
-	# shellcheck disable=SC2039
-	/bin/echo -e "\e[104m\e[97m[INFO]\e[49m\e[39m ${*}"
+	printf "\e[104m\e[97m[INFO]\e[49m\e[39m %s\n" "$@"
 }
 
 WARNING() {
-	# shellcheck disable=SC2039
-	/bin/echo >&2 -e "\e[101m\e[97m[WARNING]\e[49m\e[39m ${*}"
+	printf "\e[101m\e[97m[WARNING]\e[49m\e[39m %s\n" "$@" >&2
```

Comment:
See above.

Also, I personally prefer `>&2 printf` (redirect first) in that case as it makes it more clear immediately we are printing on stderr.

Reply:
I had to learn that syntax, because it is not intuitive. It is like shell is breaking command parameters to extract its own instructions from here. I was more comfortable thinking that ">&2" is some kind of chaining option that only comes at the end of a command, and also marks the end.